pom.xml

```diff
@@ -557,6 +558,30 @@
             <artifactId>jakarta.xml.bind-api</artifactId>
             <version>${xml.bind.api.version}</version>
             <scope>compile</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.sun.activation</groupId>
+                    <artifactId>jakarta.activation</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.angus</groupId>
+            <artifactId>angus-activation</artifactId>
+            <version>${jakarta.angus.activation.version}</version>
+            <scope>compile</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>jakarta.activation</groupId>
+                    <artifactId>jakarta.activation-api</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>jakarta.activation</groupId>
+            <artifactId>jakarta.activation-api</artifactId>
+            <version>${jakarta.activation.api.version}</version>
+            <scope>compile</scope>
```

Comment:
Why do we need the exclusion and extra dependency, when 2.1.0 is imported already from angus-activation? https://github.com/eclipse-ee4j/angus-activation/blob/92d6c661cd79bcbc69f0cd5ccdf45da4ffdb048a/pom.xml#L36

Reply:
@coheigea  For the api dependency, I want to import it directly without a transitive dependency through angus-activation.

Reply:
@jimma Why?

Reply:
@coheigea There are 2 reasons :  
1. It's clear to know the project uses api activation 2.1.0
2. We can replace with another alternative implementation in the future.